### PR TITLE
Removed duplicate declaration of locate_config_value method.

### PR DESCRIPTION
**Pull request by Aliasgar16**

### CODE REVIEW

File: lib/chef/knife/bootstrap_windows_base.rb [hunk 1]
```diff
@@ -362,11 +362,6 @@ def bootstrap_bat_file
         @bootstrap_bat_file ||= "\"%TEMP%\\bootstrap-#{Process.pid}-#{Time.now.to_i}.bat\""
       end
 
-      def locate_config_value(key)
```
Comment (mwrock) [inline lib/chef/knife/bootstrap_windows_base.rb]: Nice!

File: lib/chef/knife/winrm_base.rb [hunk 2]
```diff
@@ -117,7 +117,6 @@ def self.included(includer)
       def locate_config_value(key)
         key = key.to_sym
         value = config[key] || Chef::Config[:knife][key] || default_config[key]
-        Chef::Log.debug("Looking for key #{key} and found value #{value}")
```
Comment (mwrock) [inline lib/chef/knife/winrm_base.rb]: I'd keep this as it can be helpful for troubleshooting.

**Comment by Aliasgar16** [comment]
@mwrock the method `locate_config_value` has been declared at two places under `knife-windows` https://github.com/chef/knife-windows/blob/master/lib/chef/knife/winrm_base.rb#L117 and https://github.com/chef/knife-windows/blob/master/lib/chef/knife/bootstrap_windows_base.rb#L365.
You are right that removing this method from `bootstrap_windows_base` will break bootstrap ssh. We should ideally have this method defined only once.

**Comment by mwrock** [comment]
Totally agree we should not have 2. In fact, we really shouldn't even have 1 :) (see #204).

File: lib/chef/knife/winrm_base.rb (hunk 2 above)
Comment (adamedx) [inline lib/chef/knife/winrm_base.rb]: Yes, debug is ok, that's not the default, so we should keep this.

**Comment by mwrock** [comment]
@Aliasgar16 consider creating a `KnifeWindowsBase` module that defines `locate_config_value` and would replace the other 2.

**Comment by adamedx** [comment]
@mwrock, thoughts on this after the updates?

**Comment by mwrock** [comment]
LGTM :+1:

**Pull request closed without merging**